wagtail/admin/templates/wagtailadmin/generic/listing_results.html

```diff
@@ -49,21 +49,6 @@ <h2 role="alert">
     {% endblock %}
 {% else %}
     <div class="nice-padding w-mt-8">
-        {% fragment as no_results_message %}
-            {% if not verbose_name_plural %}
-                {% trans "There are no results." as no_results_text %}
-            {% elif is_searching or is_filtering %}
-                {% blocktrans trimmed with model_name=verbose_name_plural asvar no_results_text %}
-                    No {{ model_name }} match your query.
-                {% endblocktrans %}
-            {% else %}
-                {% blocktrans trimmed with model_name=verbose_name_plural asvar no_results_text %}
-                    There are no {{ model_name }} to display.
-                {% endblocktrans %}
-            {% endif %}
-            {{ no_results_text|capfirst }}
```

Comment:
Looks like this `capfirst` has got lost in the move.

Reply:
Oops, thanks! Rebased and amended as 9d8507b